packages/core/src/storage/libsql/index.ts

```diff
@@ -18,6 +18,17 @@ function safelyParseJSON(jsonString: string): any {
   }
 }
 
+function validateIdentifier(name: string, kind = 'identifier') {
```

Comment:
probably a good idea to put this in @mastra/core/utils or in @mastra/core/storage/utils

Reply:
Moved this function to @mastra/core/utils